src/utils/sources-tree/types.js

```diff
@@ -16,3 +16,34 @@ export type Node = {
   path: string,
   contents: SourceRecord | Array<Node>
 };
+
+/**
+ * Props are exported as property type for this.props and NextProps
+ * @memberof utils/sources-tree
+ * @static
+ */
+export type Props = {
```

Comment:
I think these types should still live in the component file. We can export them if you wish so that they can be used in the util...

This is one of the reasons though that i think it's simpler to pass some specific params to the utils